zerver/lib/actions.py

```diff
@@ -1298,10 +1327,16 @@ def bulk_add_subscriptions(streams, users):
     subs_to_add = [] # type: List[Tuple[Subscription, Stream]]
     for (user_profile, recipient_id, stream) in new_subs:
         color = pick_color_helper(user_profile, subs_by_user[user_profile.id])
+        # Give stream admin permission is stream is created by user.
+        if current_user == user_profile:
+            permissions = 15
```

Comment:
Avoid magic numbers. Replace the right side of the number with an explicit construction. Take a look at https://github.com/zulip/zulip/blob/master/zerver/models.py#L1198 for how the UserMessage model handles flags